### DISCUSSION TITLE
play application skywalking

### DISCUSSION THREAD

**Discussion by someone-1** [playframework/playframework#11669] [Q&A]
I have a play 2.5 application

I am using skywalking java agent to instrument the application.
the skywalking agent supports 2.6+ 

the skywalking agent is adding a filter method to `EnabledFilters` 

I don't see the EnabledFilters class in play2.5 is there anything similar for 2.5.x version where I can add filter?

**Comment by mkurz** [comment]
Why not just upgrade to Play 2.6 (and then 2.7 followed by 2.8)?
